src/components/configurable-items/configurable-items.stories.js

```diff
@@ -38,73 +38,63 @@ const defaultConfigurableItemsData = [
     key: "3",
   },
 ];
-const store = new Store({
-  configurableItemsData: cloneDeep(defaultConfigurableItemsData),
-});
-const handleDrag = (draggedIndex, hoveredIndex) => {
-  const configurableItemsData = store.get("configurableItemsData");
-  const draggedItem = configurableItemsData[draggedIndex];
 
-  configurableItemsData[draggedIndex] = configurableItemsData[hoveredIndex];
-  configurableItemsData[hoveredIndex] = draggedItem;
+export const Default = () => {
```

Comment:
Left this comment regarding the drag and drop.
https://github.com/Sage/carbon/pull/3510#issuecomment-747532017

Reply:
Apparently one of the Storybooks story containers uses `transform: scale(1);` turning that off fixes the issue but I'm not sure if it's a good idea to do that though :| 
![Zrzut ekranu 2020-12-18 o 15 50 56](https://user-images.githubusercontent.com/20651022/102627868-22f79100-4149-11eb-9f38-eb46de83a4a2.png)